pointcept/engines/test_registry.py

```diff
@@ -0,0 +1,3 @@
+from pointcept.utils.registry import Registry
```

Comment:
There was a circular dependency issue when running `python tools/test.py` , so I moved the imports up the tree